content/docs/deployment/csmoperator/modules/authorization.md

```diff
@@ -123,14 +119,11 @@ To enable reporting of trace data with [Zipkin](https://zipkin.io/), use the `cs
     kubectl create -f <CERTIFICATE FILE>
     ```
 
->__Note__:  
-> - If you are installing CSM Authorization in a different namespace than `authorization`, edit the `namespace` field in this file to your namespace.
-
 ### Verify Installation of the CSM Authorization Proxy Server
 Once the Authorization CR is created, you can verify the installation as mentioned below:
 
   ```bash
-  kubectl describe csm/<name-of-custom-resource> -n authorization
```

Comment:
What is this change for? Is there another ticket related to this? And also the above deletions.

Reply:
In the other PR we state that "Use the default namespace “authorization” for installing Authorization using CSM Operator." https://github.com/dell/csm/pull/1211/files. Now if this statements i true "CSM Operator does not support dynamic namespaces for Authorization. Despite successful installation in a namespace other than "authorization", errors may arise during volume creation. Use the default namespace “authorization” for installing Authorization using CSM Operator." why not have that reflected as such instead of <namespace> below?

Reply:
Earlier https://github.com/dell/csm/issues/1205 was targeted for CSM 1.10.1 and hence this was added here. This PR is simply reverting that change.